.travis/install.sh

```diff
@@ -3,4 +3,4 @@
 script_path=$(python -c "import os; import sys; print(os.path.realpath(sys.argv[1]))" "${BASH_SOURCE[0]}")
 source "${script_path%/*}/setup.sh"
 
-time ONNX_NAMESPACE=ONNX_NAMESPACE_FOO_BAR_FOR_CI pip install -v .
+time ONNX_NAMESPACE=ONNX_NAMESPACE_FOO_BAR_FOR_CI pip install -v .[mypy]
```

Comment:
I prefer to keep here the same (because we want to test whether the "standard" installation works), and then after all the tests have passed, we install mypy and do type checking (like what we do for using flake8 to lint).